src/org/recompile/freej2me/Libretro.java

```diff
@@ -211,7 +211,7 @@ public void run()
 									}
 									else
 									{
-										Mobile.getPlatform().pointerDragged(mousey, mousex);
+										Mobile.getPlatform().pointerDragged(-mousey+lcdWidth, mousex);

```

Comment:
This isn't important, but `lcdWidth-mousey` makes the intent more obvious.

Reply:
Oh that was because i wrote the properties of doing a 90 deg counterclockwise rotation, then corrected the pointer position it by adding the width, perhaps the translation to code was a bit too direct. But yes, that recommendation makes it much easier to read. I'll be updating it later.